bundles/org.openhab.binding.zm/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:zm:server">
+		<parameter-group name="url-info">
+			<label>Zoneminder URL Information</label>
+		</parameter-group>
+		<parameter-group name="config-info">
+			<label>Bridge Configuration</label>
+		</parameter-group>
+		<parameter-group name="auth-info">
+			<label>Authentication Information</label>
+		</parameter-group>
+		<parameter name="refreshInterval" type="integer" min="2" required="false" unit="s" groupName="config-info">
+			<label>Refresh Interval (Normal)</label>
+			<description>Interval in seconds at which monitor status is updated</description>
+			<default>5</default>
+		</parameter>
+		<parameter name="discoveryEnabled" type="boolean" required="false" groupName="config-info">
+			<label>Discovery Enabled</label>
+			<description>Enable/disable automatic discovery</description>
+			<default>true</default>
+		</parameter>
+		<parameter name="discoveryInterval" type="integer" min="60" required="false" unit="s"
+			groupName="config-info">
+			<label>Monitor Discovery Interval</label>
+			<description>Specifies time in seconds in which the binding will attempt to discover monitors</description>
+			<default>300</default>
+		</parameter>
+		<parameter name="defaultAlarmDuration" type="integer" unit="s" required="false" groupName="config-info">
+			<label>Default Alarm Duration</label>
+			<description>Duration in seconds after which the alarm will be turned off</description>
+			<default>60</default>
+		</parameter>
+		<parameter name="defaultImageRefreshInterval" type="integer" required="false" groupName="config-info">
+			<label>Default Image Refresh Interval</label>
+			<description>Interval in seconds at which monitor image snapshot will be updated</description>
+		</parameter>
+		<parameter name="host" type="text" required="true" groupName="url-info">
+			<label>Server</label>
+			<description>Zoneminder server name or IP address</description>
+			<context>network-address</context>
+		</parameter>
+		<parameter name="portNumber" type="text" required="false" groupName="url-info">
```

Comment:
You might want to specify min/max values.
```suggestion
		<parameter name="portNumber" type="integer" required="false" groupName="url-info">
```

Reply:
Good idea. Done.